pages.bn/android/pkg.md

```diff
@@ -0,0 +1,24 @@
+# pkg
+
+> Termux এর জন্য একটি প্যাকেজ ব্যবস্থাপনা ইউটিলিটি।
+> আরও তথ্য পাবেন: <https://wiki.termux.com/wiki/Package_Management>.
+
+- সমস্ত ইনস্টল করা প্যাকেজ আপগ্রেড করুন:
+
+`pkg upgrade`
+
+- একটি প্যাকেজ ইনস্টল করুন:
+
+`pkg install {{প্যাকেজ}}`
```

Comment:
```suggestion
- একটি নির্দিষ্ট প্যাকেজ ইনস্টল করুন:

`pkg install {{প্যাকেজ}}`
```

Optional fix (error source: original page): unexpected `package` term, `specific` expected before it.